ui-v2/yarn.lock

```diff
@@ -610,8 +610,8 @@
   resolved "https://registry.yarnpkg.com/@hashicorp/consul-api-double/-/consul-api-double-2.0.1.tgz#eaf2e3f230fbdd876c90b931fd4bb4d94aac10e2"
 
 "@hashicorp/ember-cli-api-double@^1.3.0":
-  version "1.6.1"
-  resolved "https://registry.yarnpkg.com/@hashicorp/ember-cli-api-double/-/ember-cli-api-double-1.6.1.tgz#c45efe045da971dcb0915f182cdc4a75929a7ade"
+  version "1.7.0"
```

Comment:
> I think this should be coupled with a change to package.json

...unless you call a `yarn upgrade` to upgrade a dependency according to its specified semantic version in the package.json, which is what I did here.

I'm using semantic versioning to say I want backwards compatible changes and bug fixes only. So I don't have to change the `package.json` to upgrade. I can choose to upgrade a single dependency when I want to (not automatically) by using `yarn upgrade @hashicorp/ember-cli-api-double` which took me from 1.6.1 to 1.7.0, and changed the `yarn.lock` only, so if any other dev's do an install they get the exact same locked version as me.

Reply:
Ah yeah, I usually update by editing the package.json and then run `yarn install` mostly because I'm paranoid and I like to be able to see the version there rather than scanning through the lock file.